server/controllers/interviewController.js

```diff
@@ -6,19 +6,33 @@ const { secretKey } = process.env;
 const jwt = require('jsonwebtoken');
 const { User } = require('../models/User.js');
 
-module.exports = {
-  cInterview: async (req, res) => {
-    const interview = await interviewModel.createInterview(req);
-    res.send({ interview });
-  },
-  endInterview: async (req, res) => {
-    const id = req.params.id;
-    if (!mongoose.Types.ObjectId.isValid(id))
-      res.status(400).json({ msg: 'Invalid ID' });
+const cInterview = async (req, res) => {
```

Comment:
Can we use a more descriptive name than `cInterview`? Maybe `startInterview` since we have an endInterview